vaadin-infinite-scroller.html

```diff
@@ -341,21 +342,27 @@
         });
       }
     },
+    /**
+     * Polymer 2 templatizer compatible wrapper for the _forwardParentProp
+     */
+    _forwardHostPropV2: function(prop, value) {
```

Comment:
This is the updated method name used by the Polymer Templatizer in Polymer 2 - it's the same functionality as th method that was previously handled by `_forwardParentProp` hence this wrapper.